src/Magnum/Platform/Sdl2Application.cpp

```diff
@@ -275,6 +275,52 @@ void Sdl2Application::setWindowTitle(const std::string& title) {
     #endif
 }
 
+#ifndef CORRADE_TARGET_EMSCRIPTEN
+void Sdl2Application::setCursor(Cursor cursor) {
+    CORRADE_INTERNAL_ASSERT(UnsignedInt(cursor) < Containers::arraySize(_cursors));
+
+    _cursor = cursor;
+
+    if(cursor == Cursor::Hidden) {
+        SDL_ShowCursor(SDL_FALSE);
+        return;
```

Comment:
If I set the cursor to `HiddenLocked` and then to `Hidden`, it would stay locked. So I guess here it should also do

```cpp
        SDL_SetWindowGrab(_window, SDL_FALSE);
        SDL_SetRelativeMouseMode(SDL_FALSE);
```